MEETING_SCHEDULE.md

```diff
@@ -0,0 +1,20 @@
+# Community Meeting Schedule
+
+## Office Hours
+
+We hold Ambassador Developer Office Hours once a week on Thursdays.  These sessions are a way to connect in person to share updates and listen to feedback from the community. 
+
+For the latest Ambassador Developer Office Hours information, [visit the website](https://www.getambassador.io/developer-office-hours/).
+
+
+**Zoom Meeting Link**: https://us02web.zoom.us/j/86920930400
+
+**Recordings**: http://bit.ly/AmbassadorOfficeHoursRecordings
+
+## Contributors Meeting
+
+The Telepresence Contributors Meeting is held on the first Wednesday of every month at 11am Eastern.  The focus of this meeting is discussion of technical issues related to development of Telepresence.
```

Comment:
Totally optional (or maybe this is going to be in the reference you mentioned later on the website), should we add links to the google calendar for these events so that it's easier for people to sign up + get reminders about instead of "oh I gotta remember every 1st wednesday"?

Reply:
We're working on a longer term plan for making it easier to sign up, etc.. For now, we want to get the information out there.